### PR TITLE
Add tailoring

### PR DESCRIPTION
**Pull request by dandersonsw**
## Changes proposed in this pull request:

- Adds tailoring file to customize CIS Audit
- Updates usg audit command to use tailoring file

## Things to check

- For any logging statements, is there any chance that they could be logging sensitive data?
- Are log statements using a logging library with a logging level set? Setting a logging level means that log statements "below" that level will not be written to the output. For example, if the logging level is set to `INFO` and debugging statements are written with `log.debug` or similar, then they won't be written to the otput, which can prevent unintentional leaks of sensitive data.

## Security considerations

The tailoring file is a list of rules generated by the usg command that anyone can get if they make a ubuntu pro account. The only reason why we change a rule from true to false is if it is something that doesn't apply to our configuration, so it isn't opening up or advertising any vulnerabilities.

### CODE REVIEW

**Review by jameshochadel — COMMENTED** [review]
It's possible I'm overlooking it, but do we have the reasons for disabling certain rules documented here?

**Comment by dandersonsw** [comment]
> It's possible I'm overlooking it, but do we have the reasons for disabling certain rules documented here?

@jameshochadel We do not have the reasons for disabling certain rules documented here. Right now they are only in the spreadsheet I created, but I could add the reasoning as comments in the tailoring file if we think they should be there, or I could put them in the readme or another doc in this repo?

File: README.md [hunk 1]
```diff
@@ -58,6 +58,20 @@ cloud.gov maintains a variety of software written in a handful of programming la
...
 If you want to iterate on a pipeline in this repository, consider pushing your changes to a topic branch. Topic branches do not have merge protection, so you will be able to iterate more quickly without getting pull requests approved. Change your `pipeline.yml` in your downstream repository to reference your topic branch instead of `main` in the `common-pipelines` resource to continuously pull in your changes. (You can consider working on a topic branch in your downstream repo as well.)
 
+## CIS Rule Customization/Tailoring
+
+We set the following CIS Rule exceptions in our `tailor.xml` file:
+
+| Rule | Name | Reason for Exception |
+| ---- | ---- | -------------------- |
+| 1.4.3 | `xccdf_org.ssgproject.content_rule_ensure_root_password_configured` | Not applicable to containers |
+| 3.5.2.9 | `xccdf_org.ssgproject.content_rule_service_nftables_enabled` | False Positive: nftables is enabled |
```
Comment (jameshochadel) [inline README.md:68]: Can we document how we determined, like the command we ran to show it?

**Review by jameshochadel — COMMENTED** [review]
Love it. Left one comment, otherwise this looks ready to ship.

File: README.md (hunk 1 above)
Comment (dandersonsw) [reply README.md]: Updated with this documentation

**Review by jameshochadel — APPROVED** [review]
Perfect. 🚀

**Pull request merged**